app/controllers/aspect_memberships_controller.rb

```diff
@@ -76,7 +76,7 @@ def create
   end
 
   rescue_from Diaspora::NotMine do
-    render :text => "You are not allowed to do that.", :status => 403
+    render :text => I18n.t('aspect_memberships.destroy.forbitten'), :status => 403
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.